Random Users/View Controllers/UserDetailViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  UserDetailViewController.swift
+//  Random Users
+//
+//  Created by Rob Vance on 9/11/20.
+//  Copyright © 2020 Erica Sadun. All rights reserved.
+//
+
+import UIKit
+
+class UserDetailViewController: UIViewController {
+    
+    //MARK: - Properties -
+    var randomUserAPICon: RandomUsersApiController?
+    var user: User?
+    
+    //MARK: - IBOutlets -
+    @IBOutlet weak var largeImageView: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var phoneNumberLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updateViews()
+    }
+    
+    func updateViews() {
+        nameLabel.text = user?.name
+        phoneNumberLabel.text = user?.phone
+        emailLabel.text = user?.email
+        
+        do {
+            guard let url = user?.largeImage else { return }
+            let data = try Data(contentsOf: url)
+            let image = UIImage(data: data)
```

Comment:
⚠️ DO NOT EVER DO THIS IN AN ACTUAL APP!!! I know this is a bit extreme with the all caps, but this is really, really, bad. This is the exact opposite of what the point of this concurrency sprint is all about. Data(contentsOf:) runs on the main thread, where the UI runs. Therefore, it blocks the UI while fetching the image. This is not at all what we want. 

Fetching this image in your APIController would have been acceptable, as would using an operation. Please change this in your app at some point (when you have time, don't take away from what you are learning now).